content/articles/how-sweet-is-kubernetes/index.md

```diff
@@ -0,0 +1,254 @@
+**HOW SWEET IS KUBERNETES?**
+
+![kubernetes.jpeg](kub.jpeg)
+
+What was your first impression when you read Kubernetes?
+
+I bet you were already thinking,
+*“Oh no, here comes another tech buzzword. Already sounds like a difficult concept.”*
+
+Well, you are not so right. But let’s dive in!
+
+At first, it would be better to understand that we would be using the Fleetman application to explain the concept of Kubernetes.
+
+NB: You can clone the GitHub repository used to describe this tutorial on:
+
+[Deploying fleetman with kubernetes](https://github.com/wizardcalidad/deploying_fleetman_application_with_kubernetes)
+
+The Fleetman Application is built for  a company that needs to track the location of their trucks as they move around the city of London. Those trucks are each equipped with Global Positioning System(GPS) and periodically report their position to the central server. It is a simple microservice architecture with little to no complexity, it is not your day-to-day hello world application. We are using some pre-built images on DockerHub.com under the Richard Chesterwood repository.
+
+![Fleetman microservice](fleetman.png)
+
+**OVERVIEW OF FLEETMAN APPLICATION**
+
+Obviously, we are not discussing fleetman’s application here, instead, we will be using fleetman’s application to better understand our content, do not be discombobulated if I did not explain the flow of the processes in the YAML files below in full.
+
+
+Microservices of course obey the SOLID principle, the first being the Single Responsibility Principle, each and every part of the architecture should do just one thing.
+
+The Position Simulation only simulates the positions of the truck at any point in time and that is its only job, once it reads the position, it hands it over to the Message Queue, the essence of a queue in any architecture of microservice is to avoid loss of data. Instead of making the position simulator and the tracker tightly coupled, a queue is introduced in between to avoid loss of data at any instance. The queue receives the data, keeping it waiting for the request from the position tracker. The queue implemented here is ActiveMQ, it is also utilized as a docker container. The position tracker gets the information from the queue and performs a lot of calculations with it, like calculating the speed a certain truck has covered. API Gateway is the single entry point to the entire application, its job is to match all requests incoming to underlying microservices.
+
+**WHAT KUBERNETES IS NOT**
+
+Kubernetes is not an alternative to Docker. It is something more.
+
+**WHAT KUBERNETES IS**
```

Comment:
Have this come before what Kubernetes is not.

Reply:
done